warehouse/query-core/src/test/java/datawave/query/index/lookup/IndexStreamComparatorTest.java

```diff
@@ -154,32 +134,20 @@ private ScannerStream s3() {
         return buildScannerStream("F3", "c", PRESENT);
     }
 
-    private ScannerStream ignored() {
-        return buildScannerStream("F3", "c", IGNORED);
-    }
-
-    private ScannerStream unindexed() {
-        return buildScannerStream("F4", "d", UNINDEXED);
-    }
-
     private ScannerStream absent() {
         return buildScannerStream("F5", "e", ABSENT);
     }
 
     private ScannerStream delayed() {
-        return buildScannerStream("F6", "f", DELAYED_FIELD);
-    }
-
-    private ScannerStream unknown() {
-        return buildScannerStream("F7", "g", UNKNOWN_FIELD);
+        return buildScannerStream("F6", "f", DELAYED);
     }
 
     private ScannerStream exceededTerm() {
-        return buildScannerStream("F8", "h", EXCEEDED_TERM_THRESHOLD);
+        return buildScannerStream("F8", "h", DELAYED);
     }
 
     private ScannerStream exceededValue() {
-        return buildScannerStream("F9", "i", EXCEEDED_VALUE_THRESHOLD);
+        return buildScannerStream("F9", "i", PRESENT);
```

Comment:
why wouldn't an EXCEEDED_VALUE_THRESHOLD be converted to DELAYED?

Reply:
Because we cheat. Exceeded value and exceeded or markers are given special treatment where we construct a fake stream of shards. Now that I think about it...we should do the same thing for bounded range markers...